pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tqdm_publisher"
+version="0.1.0"
+authors = [
+  { name="Garrett Flynn", email="garrett.flynn@catalystneuro.com" },
+  { name="Cody Baker", email="cody.baker@catalystneuro.com" },
+]
+description = "Subscribe to TQDM progress bars with arbitrary functions."
```

Comment:
Can you expand on this and why it's useful / make it clear how it's different from TQDM?

Reply:
Shouldn't this be quite short, where we let the README do the rest of the work (e.g. [TQDM pyproject.toml](https://github.com/tqdm/tqdm/blob/master/pyproject.toml))

Reply:
Yes, it should be short (one line)

But it should also be a clear and concise description too; 'arbitrary functions' could mean anything, 

"Extends the base tqdm progress bar with a subscribe method which can be used to expose updates to external (potentially non-Python) processes."

Or something of that nature

Reply:
Done. Thanks for the clarification.